### PR TITLE
net-im/ejabberd: update usermod handling

### PR DESCRIPTION
**Pull request by ConiKost**
Signed-off-by: Conrad Kostecki <conikost@gentoo.org>

@Flowdalic Please review.

### CODE REVIEW

File: acct-user/ejabberd/ejabberd-2.ebuild [hunk 1]
```diff
@@ -7,8 +7,14 @@ inherit acct-user
 
 DESCRIPTION="User for net-im/ejabberd"
 
-ACCT_USER_ID=114
-ACCT_USER_GROUPS=( ${PN} )
-ACCT_USER_HOME=/var/lib/${PN}
+ACCT_USER_ID="114"
+ACCT_USER_GROUPS=( "ejabberd" )
```
Comment (Flowdalic) [inline acct-user/ejabberd/ejabberd-2.ebuild:11]: I'd prefer to keep this as is.

File: net-im/ejabberd/ejabberd-21.12-r2.ebuild [hunk 2]
```diff
@@ -56,7 +56,7 @@ DEPEND=">=dev-lang/erlang-19.3[odbc?,ssl]
 	stun? ( >=dev-erlang/stun-1.0.47 )
 	zlib? ( >=dev-erlang/ezlib-1.0.10 )"
 RDEPEND="${DEPEND}
-	acct-user/ejabberd
+	acct-user/ejabberd[pam?]
```
Comment (Flowdalic) [inline net-im/ejabberd/ejabberd-21.12-r2.ebuild:59]:
```suggestion
	>=acct-user/ejabberd-2[pam?]
```

and if it is just to denote when the pam USE flag was introduced to the acct-user/ejabberd ebuild.

File: net-im/ejabberd/ejabberd-22.05-r2.ebuild [hunk 3]
```diff
@@ -60,7 +60,7 @@ DEPEND="
 "
 RDEPEND="
 	${DEPEND}
-	>=acct-user/ejabberd-1
+	>=acct-user/ejabberd-1[pam?]
```
Comment (Flowdalic) [inline net-im/ejabberd/ejabberd-22.05-r2.ebuild:63]:
```suggestion
	>=acct-user/ejabberd-2[pam?]
```

as above

**Review by Flowdalic — CHANGES_REQUESTED** [review]
Thanks @ConiKost Nice find. +1

I only have some minor remarks.

File: acct-user/ejabberd/ejabberd-2.ebuild (hunk 1 above)
Comment (ConiKost) [reply acct-user/ejabberd/ejabberd-2.ebuild]: Done.

File: net-im/ejabberd/ejabberd-21.12-r2.ebuild (hunk 2 above)
Comment (ConiKost) [reply net-im/ejabberd/ejabberd-21.12-r2.ebuild]: Done.

File: net-im/ejabberd/ejabberd-22.05-r2.ebuild (hunk 3 above)
Comment (ConiKost) [reply net-im/ejabberd/ejabberd-22.05-r2.ebuild]: Done.

**Pull request closed without merging**